### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by ungoldman**
Hi @zeke 👋

Thought it might be a good idea to add a description, installation instructions, and license to the readme.

### CODE REVIEW

**Comment by ungoldman** [comment]
Preview: https://github.com/ungoldman/release-notifier/blob/docs/README.md

File: README.md [hunk 1]
```diff
@@ -2,4 +2,19 @@
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/release-notifier/release-notifier.svg)](https://greenkeeper.io/)
 
-Source code for GitHub app
+A [probot](https://github.com/probot/probot) that leaves helpful comments on your pull requests when they land in a release.
```
Comment (gr2m) [inline README.md]: maybe capitalize "Probot", other than that this is great :)

**Comment by zeke** [comment]
Thanks, Nate. This is awesome. Can you also add a note about how this is designed for use with projects that are using GitHub Releases and Semantic Versioning?

**Comment by ungoldman** [comment]
OK, updated! Thanks for the review @zeke & @gr2m 😁

**Comment by zeke** [comment]
☝️ :)

**Pull request merged**